src/main/java/org/liquidengine/legui/component/misc/listener/textarea/TextAreaViewportScrollListener.java

```diff
@@ -40,14 +42,15 @@ public void process(ScrollEvent event) {
         TextArea textArea = (TextArea) event.getTargetComponent().getParent();
 
         TextAreaField textAreaField = textArea.getTextAreaField();
-        float maxTextWidth = Math.max(textAreaField.getMaxTextWidth() +
-                        textAreaField.getStyle().getPaddingLeftF() +
-                        textAreaField.getStyle().getPaddingRightF(),
-                textArea.getViewportSize().x);
-        float maxTextHeight = Math.max(textAreaField.getMaxTextHeight() +
-                        textAreaField.getStyle().getPaddingTopF() +
-                        textAreaField.getStyle().getPaddingBottomF(),
-                textArea.getViewportSize().y);
+        Vector4f p = getPaddingV4(textAreaField, textAreaField.getStyle());
```

Comment:
Rename variable to `padding` for clarity

Reply:
Done :)